src/EnergyPlus/CMakeLists.txt

```diff
@@ -991,11 +992,37 @@ if(BUILD_PACKAGE)
     # we'll want to grab the standard lib for python plugins
     # TODO: I don't think we want to quote the generator expression
     install(DIRECTORY "$<TARGET_FILE_DIR:energyplus>/python_lib/" DESTINATION "./python_lib")
-    if(WIN32 AND PYTHON_CLI)
-      # on Windows, with Build Package, and also Link With Python, we can also drop in shortcuts to the new auxiliary CLI
-      # NOTE: The install command COMPONENTS should line up so that they are run at the same packaging step
-      install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE:energyplus>\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk\" -Arguments \"auxiliary eplaunch\")" COMPONENT Auxiliary)
-      install(FILES $<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk DESTINATION "./" COMPONENT Auxiliary)
+    if(PYTHON_CLI)
+      if(WIN32)
```

Comment:
For Python CLI on Windows, we now build a tool called windows_gui_launcher.exe, whose role is to execute a command line tool without opening a terminal so that it can launch a GUI nicely.  We also create shortcuts in the EnergyPlus install root so that the user can find them there.   Perhaps I should move the IDFVersionUpdater into the PreProcess folder?